### PR TITLE
Reverted post design to old card

### PR DESCRIPTION
**Pull request by mbos2**
Reverted post design to old card
Updated services to fetch user data on explore
Updated user route to fetch by user id, not by logged in id

### CODE REVIEW

File: src/app/core/services/auth.service.ts [hunk 1]
```diff
@@ -31,14 +31,11 @@ export class AuthService {
...
 
-    await this.supabaseClient.from('users').insert([{ id: user?.id }]);
-    await this.supabaseClient.from('user_profile').insert([
-      {
-        user_id: user?.id,
-        first_name: credentials.firstname,
-        last_name: credentials.lastname,
-      },
-    ]);
+    await this.supabaseClient.from('users').insert([{
```
Comment (Knochenmark) [inline src/app/core/services/auth.service.ts:34]: we should improve our error handling here, but don't worry, I will take care of it

File: src/app/core/services/database.service.ts [hunk 2]
```diff
@@ -19,13 +19,16 @@ export class DatabaseService {
 
   async getUserProfile(id: any) {
     return await this.supabaseClient
-      .from('user_profile')
-      .select('first_name, last_name, about, user_image_path')
-      .eq('user_id', id);
+      .from('users')
+      .select('id, first_name, last_name, about, profile_image')
+      .eq('id', id);
```
Comment (Knochenmark) [inline src/app/core/services/database.service.ts]: can you fix naming and typing i.e. `userId: string` for example

File: src/app/modules/explore/explore.component.ts [hunk 3]
```diff
@@ -41,9 +40,8 @@ export class ExploreComponent implements OnInit {
   }
 
   async getPosts() {
-    const { data, error } = await this.supabaseClient
-      .from('posts')
-      .select(`id, created_at, user_id, title, description, images_path`);
+    const { data, error } = await this.db.getAllPosts();
```
Comment (Knochenmark) [inline src/app/modules/explore/explore.component.ts:43]: we need more error handling

File: src/app/modules/explore/post/post.component.styl [hunk 4]
```diff
@@ -1,37 +1,28 @@
-.container
-  position relative
+.card
+  max-width 400px
```
Comment (Knochenmark) [inline src/app/modules/explore/post/post.component.styl]: shouldn't the cards be capped by the masonry columns already?

File: src/app/modules/new-post/new-post.component.ts [hunk 5]
```diff
@@ -50,14 +46,14 @@ export class NewPostComponent {
...
-      this.tags.push({ name: value.trim() });
+      this.tags.push('#' + value.trim());
     }
     if (input) {
       input.value = '';
     }
   }
 
-  removeTag(tags: Tag): void {
+  removeTag(tags: string): void {
```
Comment (Knochenmark) [inline src/app/modules/new-post/new-post.component.ts]: this is probably just a single tag? the param is a bit misleading

**Review by Knochenmark — CHANGES_REQUESTED** [review]

File: src/app/core/services/database.service.ts (hunk 2 above)
Comment (mbos2) [reply src/app/core/services/database.service.ts]: done

**Pull request merged**